__tests__/helper-functions.js

```diff
@@ -1,13 +1,13 @@
 const { gql } = require('@apollo/client');
 const User = require('../modules/user/user.model');
 const { ApolloServer } = require('apollo-server-express');
-const client = require('../utils/apollo-test-client');
 const config = require('../app');
 const { createTestClient } = require('apollo-server-testing');
 const bcrypt = require('bcryptjs');
 
 const adminLogin = async user => {
```

Comment:
Please, rename it to the `loginAdmin`

Reply:
Do we really need this kind of method at all? Currently, when we use the testing server we already provide the `context` with specific users inside it. So, fetching token doesn't make a lot of sense to us. Do I miss something?

Reply:
solved